packaging/greenboot/microshift-running-check.sh

```diff
@@ -3,8 +3,8 @@ set -e
 
 SCRIPT_NAME=$(basename $0)
 SCRIPT_PID=$$
-OCGET_CMD="microshift get"
-OCGET_OPT="--kubeconfig /var/lib/microshift/resources/kubeadmin/kubeconfig --no-headers"
+OCGET_CMD="oc get --kubeconfig /var/lib/microshift/resources/kubeadmin/kubeconfig"
+OCGET_OPT="--no-headers"
```

Comment:
Why did we move `--kubeconfig` as part of the command?

Reply:
Any command using `oc` must have that option because the default KUBECONFIG is unset.
I thought of including that into `OCGET_OPT` but the `get --raw` does print any headers, and even though the option does not make the command fail it is redundant/confusing.